packages/types/src/span.ts

```diff
@@ -132,9 +132,7 @@ export interface Span extends SpanContext {
    * Creates a new `Span` while setting the current `Span.id` as `parentSpanId`.
    * Also the `sampled` decision will be inherited.
    */
-  startChild(
-    spanContext?: Pick<SpanContext, Exclude<keyof SpanContext, 'spanId' | 'sampled' | 'traceId' | 'parentSpanId'>>,
-  ): Span;
+  startChild(spanContext?: Pick<SpanContext, Exclude<keyof SpanContext, 'sampled' | 'traceId' | 'parentSpanId'>>): Span;
```

Comment:
I think this was missed in https://github.com/getsentry/sentry-javascript/pull/6028?

Reply:
yes 🤦